src/firefox/preferences.js

```diff
@@ -136,8 +136,13 @@ export function coerceCLICustomPreference(
 
   for (const pref of cliPrefs) {
     const prefsAry = pref.split('=');
+
+    if (prefsAry.length < 2) {
+      throw new UsageError(`Incomplete custom preference: "${pref}"`);
```

Comment:
This might be more helpful with an example, like "Incomplete custom preference (expected key=value): ..."